### DISCUSSION TITLE
[Feedback] Frustrating experience: "Upload proof of your academic status"

### DISCUSSION THREAD

**Discussion by Gabgobie** [community/community#138383] [GitHub Education]
### Hi everyone,

Hi, I hope you are all doing well.

I am writing to share my frustration with the upload of proof of my academic status in hopes of giving some insight and improving the process for my fellow students.

My university is providing PDF documents as proof of my enrollment. These certificates also contain a code as well as a link where they can be validated. Unfortunately they cannot be uploaded and I have to take a picture instead. The rationale behind this seems to be the prevention of fraudulent applications according to other discussions I have seen. That doesn't really apply to me since there are the validation codes but well.. whatever. I'll just use my phone and take a picture of my screen.

Except that doesn't work either. My phone (Apple 11th Gen) is certainly not the most recent but it has a good camera and still does its job. GitHub will ask for permission to access my camera, which I grant and my face shows up on the screen. Wait. My face? Why isn't it using the rear camera? But there is a drop down menu so I can choose another one of the seven total options. I thought. But nothing except the front camera works. Now I am stuck trying to take a picture of the document with my front camera that still has at least somewhat legible text. An almost impossible job as it seems as there seems to be additional compression applied before the picture is uploaded or at least the preview is looking horrible and I am having a hard enough time decyphering the text and I expect even worse from an OCR algorithm that is supposed to validate the document for me.

I would wholeheartedly appreciate if it was made possible to upload PDF documents as proof of enrollment, especially if they have some kind of validation mechanism, be it a cryptographic signature, a valiadtion code or something else entirely.

Best,
Gab

PS. My application seems to have been approved immediately and I should wait for my benefits to arrive so I am not facing issues but this is mainly about the process (and the broken camera settings). Edit: I am also quite impressed by that OCR algorithm.

**Comment by itisrude** [comment]
Hey Gab,

Man, that does sound frustrating! I can understand them trying to avoid fraudulent uploads, but not being able to upload a PDF-especially with codes for validation-is overkill. And that thing with the camera is weird, too-why not let him use the rear camera? That would really help in capturing quality and readability.

For now, perhaps you can use a screenshot of the PDF on your computer instead of having to take a pic with your phone, if that is easier. And yeah, if they could just allow PDFs to be uploaded with a validation code, that would save a lot of headaches.

Glad your application was still approved, though! Hopefully, they fix this for others soon enough.

**Reply by Gabgobie** [reply]
Hi,

I was actually looking into using the OBS virtual cam to get a screenshot to work, however the compression was just too much and the quality even worse than my front cam pointed at my screen or maybe I just screwed up the settings. Whichever it is, I am glad I'm through for this semester.

This part is starting to get off-topic but yesterday evening was even worse. There is a new requirement to have a profile readme that wasn't there last time I enrolled so when I tried to re-apply, there was just a red banner telling me that there are issues with my account and that I should contact support. Following the link, I was presented with the account reinstatement form and a bright red banner that my account is flagged. No trace of what it would be flagged for. I am also pretty sure it isn't (by my application going through flawlessly today) but I submitted a ticket just to be sure. Error messages that don't tell you what's wrong are not really helpful and instill fear at best.

Best,
Gab

**Reply by itisrude** [reply]
Hey Gab,

It sounds like you've really been through the wringer with this whole process! The OBS virtual cam idea was a great work-around, but I can totally understand frustration with compression issues-especially when you're trying to get just a clear shot of something as simple as a document.

New profile requirement and flagging of your account via the banner. weird. Must be super annoying, too, as you can only guess what the real problem is, since there aren't any specific error messages. Hope that clears up soon once you submit that support ticket. On a positive note, glad to hear at least your application went through smooth today!

Fingers crossed that they resolve this and give better guidance for others in the same boat. Let me know if you hear anything back from them, or if in the meantime there is anything else I can help with.

Best of luck!
